.erasim/themes/default.theme/memory.sass

```diff
@@ -0,0 +1,37 @@
+memory
+  cell
+    background: transparent
+    color: $primary-text-color
+    font-size: 12px
+    height: 24px
+    width: 80px
+  divider
+    color: $divider-color
+    width: 1px
+  address
+    background: transparent
+    color: $primary-text-color
+    font-size: 12px
+    font-weight: normal
+    margin-left: -10px
+    width: 80px
+  header
```

Comment:
`ComboBox`es look like normal text. It is not intuitive to click on them. Maybe we could change the styling a little (border?) ;)